src/store/utils.js

```diff
@@ -51,6 +51,17 @@ const getWindBackgroundColor = (windSpeedKmPerH) => {
   return windSpeedToBackgroundMapping[windSpeedNumberForMapping] || '#f36d6d';
 };
 
+const translatePrecipitationType = (precipitationTypeInEnglish, languageCode = 'uk') => {
```

Comment:
FYI there is a dedicated plugin localization
https://kazupon.github.io/vue-i18n/

Reply:
@probil, thanks.